src/monodroid/jni/android-system.h

```diff
@@ -37,10 +37,10 @@ namespace xamarin { namespace android { namespace internal
 		static std::mutex readdir_mutex;
 		static char *libmonoandroid_directory_path;
 #endif
-
-// _WIN32 is defined with _WIN64 so _WIN64 must be checked first.
-#if __SIZEOF_POINTER__ == 8 || defined (_WIN64)
-#define __BITNESS__ "64bit"
+
+// _WIN32 is defined with _WIN64 so _WIN64 must be checked first.
+#if __SIZEOF_POINTER__ == 8 || defined (_WIN64)
+#define __BITNESS__ "64bit"
```

Comment:
whitespace-only change?

Reply:
It looks like this block had CRLF and the rest of the file was LF:

![image](https://user-images.githubusercontent.com/840039/60902760-c5fa6a80-a235-11e9-8505-7cc85475b111.png)

VS Code fixed it, and I didn't notice somehow?